phpunit.xml

```diff
@@ -1,5 +1,4 @@
 <phpunit bootstrap="tests/bootstrap.php"
-         colors="true"
```

Comment:
that sounds like a phpunit bug - if it can't output colors it should ignore this. what version of phpunit do you have?